packages/@react-spectrum/dnd/docs/useDragHooks.mdx

```diff
@@ -0,0 +1,166 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import docs from 'docs:@react-spectrum/dnd';
+import listviewDocs from 'docs:@react-spectrum/list';
+import {FunctionAPI, PageDescription} from '@react-spectrum/docs';
+import {Keyboard} from '@react-spectrum/text';
+
+---
+category: Concepts
+keywords: [drag and drop, dnd]
+---
+
+# Drag and Drop
+
+This page describes how to enable drag and drop functionality for the various React Spectrum components that support it.
+
+## Introduction
+
+Drag and drop is an operation performed when a user "grabs" a UI element via their pointing device and drops it on a different location or element.
+It has become a common interaction in many modern applications due to its flexibilty across a variety of use cases but can come saddled with
+a variety of accessibility issues, including but not limited to keyboard interaction parity and difficulties mimicking drag operations with mouse emulators.
+React Spectrum components that support drag and drop operations seek to address these issues by providing full keyboard and screen reader drag and drop
+support.
+
+If you are interested in adding drag and drop support to your own components, feel free to check out the [drag and drop react-aria hooks](https://www.npmjs.com/package/@react-aria/dnd)!
+
+## API
+
+<FunctionAPI function={docs.exports.useDragHooks} links={docs.links} />
+
+## How to use
+
+Enabling drag and drop for a supported React Spectrum component differs slightly from the typical event handler prop pattern that you maybe familiar with.
+Instead of providing each event handler directly to the component, you must first import `useDragHooks` from `@react-spectrum/dnd` package and provide it with
+your desired options. `useDragHooks` will then provide you with `DragHooks` that you can pass to the component via the `dragHooks` prop.
+
+You may be wondering why we didn't simply import the drag and drop hooks within the components themselves. The reason behind this is to reduce the bundle size of
+the component for users who don't need the drag and drop support for their component.
```

Comment:
Felt like this was important to highlight since it is a bit of a departure from our typical api